CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 
 ### Minor Changes
 * Fixed a crash on iOS when creating many short-lived realms very rapidly in parallel (#653)
+* `RealmObject.IsValid` lets you know if a managed object has been deleted
```

Comment:
I think the comment _lets you know_ sounds a bit too active - how about _can be called to check_?